ignition/modules/token.js

```diff
@@ -0,0 +1,32 @@
+const { buildModule } = require("@nomicfoundation/hardhat-ignition/modules")
+
+const MAX_ACCOUNTS = 20
+const MINTED_TOKENS = 1_000_000_000_000_000n
+
+module.exports = buildModule("Token", (m) => {
+  const deployer = m.getAccount(0)
+
+  let token
+
+  if (process.env.TOKEN_ADDRESS) {
+    token = m.contractAt("TestToken", process.env.TOKEN_ADDRESS, {})
```

Comment:
```suggestion
    console.log("Using existing TestToken on address: ", process.env.TOKEN_ADDRESS)
    token = m.contractAt("TestToken", process.env.TOKEN_ADDRESS, {})
```

Let's add some logging so the deployer knows that this contract will be using the existing token. Just in case somebody, for example, forgets to clear the env. variable and can be then unpleasantly surprised.